app/Controllers/NewDonationController.php

```diff
@@ -43,7 +43,7 @@ public function handle( FunFunFactory $ffFactory, Request $request ): Response {
 				$interval,
 				$validationResult->isSuccessful(),
 				$trackingInfo,
-				$request->get( 'addressType', 'person' ),
+				$request->get( 'addressType', $ffFactory->getChoiceFactory()->getAddressType() ),
```

Comment:
Please keep the ChoiceFactory private and add a method to FunFunFactory instead. This will help removing the test because we won't have to search for ChoiceFactory usages in the code base.